scripts/jenkins/ardana/ansible/deploy-caasp.yml

```diff
@@ -0,0 +1,48 @@
+---
+
+- name: Deploy CaaSP using heat templates
+  hosts: hosts
+  gather_facts: False
+  vars:
+    caasp_image_url: "http://download.suse.de/install/SUSE-CaaSP-3-GM/SUSE-CaaS-Platform-3.0-for-OpenStack-Cloud.x86_64-3.0.0-GM.qcow2"
+    caasp_image_path: "/tmp/SUSE-CaaS-Platform-3.0-for-OpenStack-Cloud.x86_64-3.0.0-GM.qcow2"
+    image_name: "SUSE-CaaS-Platform-3.0-for-OpenStack-Cloud"
+    stack_name: "caasp-stack"
+  tasks:
+  - name: Install caasp-openstack-heat-templates
+    always_run: True
+    become: yes
+    zypper:
+      name: caasp-openstack-heat-templates
+      state: present
+
+  - name: Download CaaSP image
+    always_run: True
+    get_url:
+      url: "{{ caasp_image_url }}"
+      dest: "{{ caasp_image_path }}"
+    register: _download_caasp_image
```

Comment:
please add a permission here. (like 0600).
Else it's not enforcing both the file contents and its permissions in a folder like /tmp, which could lead to security risks.